quint/src/static/callgraph.ts

```diff
@@ -190,9 +190,6 @@ export class CallGraphVisitor implements IRVisitor {
     const importedModule = this.context.modulesByName.get(decl.protoName)
     if (importedModule) {
       this.graphAddAll(decl.id, Set([importedModule.id]))
-      decl.overrides.forEach(([_param, expr]) => {
-        this.graphAddAll(expr.id, Set([decl.id]))
-      })
```

Comment:
This is from a reverted commit, which added a hack for something we don't need anymore